CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+<!-- 
+
+- Taking inspiration from https://github.com/zowe/docs-site/blob/master/README.md
+- Commenting out sections from that until we mature and need all their process
+
+-->
+<!-- uncomment once we get to the point of welcoming feedback 
+## Providing feedback
+
+Your feedback is essential in shaping the Zowe content experience. There are several ways to provide feedback:
+
+- If you see something incorrect or confusing in the docs, or have an enhancement idea to make the docs better, you can edit a page by clicking **Edit this page** at the top of the page to [open a Pull Request](https://docs.zowe.org/stable/contribute/contributing#sending-a-github-pull-request). You can also [open a GitHub issue](https://docs.zowe.org/stable/contribute/contributing#opening-an-issue-for-the-documentation) for the documentation team by clicking **Open doc issue**.
+- You can rate the topic at the bottom of each page to tell us whether it is helpful.
+- If you have a question about docs, you can join the Zowe [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to reach out to the Zowe Doc Squad and the community.
+
+Only file issues about Zowe docs in this repository. For issues, questions, new feature requests, or enhancement ideas about a specific component or aspect of Zowe, open an issue in the [corresponding code repository](https://github.com/zowe/community#zowe-sub-projects) or ask in the [community Slack channels](https://github.com/zowe/community#slack).
+-->
+## Contributing to the project
+
+You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account. You can make content contributions in your fork, and when the changes are ready, go to your fork and create a new pull request to send the changes to us.
```

Comment:
Do we want people to Fork in this project?

Reply:
As a public project, I don't think we can deny the possibility of forking.